pkg/sfu/forwarder.go

```diff
@@ -213,7 +228,7 @@ func NewForwarder(codec webrtc.RTPCodecCapability, kind webrtc.RTPCodecType, log
 	}
 
 	if f.kind == webrtc.RTPCodecTypeVideo {
-		f.maxLayers = DefaultMaxLayers
+		f.maxLayers = VideoLayers{spatial: InvalidLayerSpatial, temporal: DefaultMaxLayerTemporal}
```

Comment:
should we default to invalid spatial layer? it used to be `DefaultMaxLayerSpatial`

Reply:
Yeah, by default turn off forwarding. It will be enabled by protocol setting if adaptive streaming is not on. Till protocol support is added, for now, setting it to max when subscribed track is created.